doc/lsst.meas.extensions.scarlet/overview.rst

```diff
@@ -0,0 +1,217 @@
+
+For *scarlet* we have chosen the *𝛘²* (inverse variance-weighted or Kaiser) co-add to initialize our morphologies.
+While this is not a truly accurate model (since each observation uses a different seeing), it gives us a fast and efficient method for generating a detection image.
+The main downfall of this choice is that our model space has a much narrower seeing than the detection image, meaning our initial models will always be "puffy."
+We have tried numerous deconvolution and approximation methods but so far we have not found any method that provides enough of an improvement to overcome the runtime required to generate the detection image.
+In other words, every method that we have come up with to generate a better detection image is more expensive computationally than just running the scarlet optimizer for a few extra iterations to converge to the same solution.
+
+Once we have fit the morphology for each component of each source, we use a least-squares algorithm to simultaneously fit the SED of each source *in scarlet model space*.
+Because we normalize the morphology, including the intensity information in the SED, this is different than the observed SED, as the PSF in each band will distribute the model flux differently.
+In this case the computational savings of the least-squares SED fit *is* significant enough to justify its use.
+
+.. note::
+    For *very* large (approx *1k* × *1k* pixel) blends the least squares fit runs into memory issues, since the size of the array to fit is roughly ``number of sources × number of bands × width × height)``.
+    In that case setting :attr:`~lsst.meas.extensions.scarlet.ScarletDeblendConfig.setSpectra` to ``False`` is recommended.
+
+.. note::
+    An interesting fact about optimization in scarlet is that the best initialization (in terms of 𝛘²) will not necessarily result in the fastest runtime or even the most optimal fit.
+    The interplay between the particular gradient descent method used and proximal operators is complicated and often certain irregularities (like sources with slightly worse positions) can result in improved optimization in terms of both runtime and log-likelihood.
+
+.. _optimization:
+
+Optimization
+^^^^^^^^^^^^
+
+In addition to being multi-band, the other main difference between *scarlet* and the single-band deblender is that the *scarlet* models are iteratively optimized using a gradient-descent-like algorithm.
+The extra time to run this optimizer is the primary reason that *scarlet* is not used on the single-visit exposures.
+We have tested a number of different algorithms over the years and so far we have achieved the best performance using proxmial ADAM, as described in `Melchior et al. 2019 <https://arxiv.org/abs/1910.10094>`_.
```

Comment:
proxmial vs proximal

Reply:
Your editing ability is much appreciated. I _never_ catch things like this.